index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="style.css" />
+  <title>Document</title>
+</head>
+
+<body>
+  <nav class="flex-nav">
+    <div class="main-head">
+      <h1>Benjamin</h1>
+    </div>
+    <div class="menu-icon">
+      <svg xmlns="http://www.w3.org/2000/svg" width="25" height="25" fill="currentColor" class="bi bi-list"
+        viewBox="0 0 16 16">
+        <path fill-rule="evenodd"
+          d="M2.5 12a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5z" />
+      </svg>
+    </div>
+    
+  </nav>
+
+  <div class="wrapper">
+
+    <div class="pri">
+      <p class="primary-text">Hey There. I’m Benjamin</p>
+    </div>
+    <div class="pri">
+      <p class="supporting-text">I'am a Software Developer</p>
+    </div>
+    <div class="pri">
+      <p class="description">I can help you build a product , feature or website Look through some of my work and
+        experience! If you like what you see and have a project you need coded,
+        don’t hestiate to contact me. </p>
+    </div>
+
+    <div class="social-media">
+      <div class="icons"><i class="fa fa-github" ></i></div>
+      <div class="icons"><i class="fa fa-linkedin-square" ></i></div>
+      <div class="icons"><i class="fa fa-twitter" ></i></div>
+      <div class="icons"><i class="fa fa-facebook-f" ></i></div>
```

Comment:
- [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)